test/integration/select_line_test.rb

```diff
@@ -24,6 +24,21 @@ class SelectLineTest < ActionDispatch::IntegrationTest
     end
   end
 
+  # TODO
+  describe 'line redirect on single line' do
+    before do
+      # TODO setup line array like so ['dc']
+      @user.line='DC'
+      # LINES=%w(DC).map(&:to_sym).freeze
+      puts @user.inspect
+      # how to change env var constants in this block?
```

Comment:
no free baskets here. on ```lines_path```, it always grabs from the env variable. Only way to get it to pass was to change ```%w(DC MD VA).map(&:to_sym).freeze``` in ```env_constants.erb``` to  ```%w(DC).map(&:to_sym).freeze```

Leaving this to you!